cloud

```diff
@@ -1911,6 +1926,7 @@ function execute_command() {
             --smtp-topic="$smtp_topic" \
             --smtp-subscription="$smtp_subscription"
         sections_run "$sections" firestore_deploy "$project"
+        sections_run "$sections" storage_deploy "$project" "$storage_bucket"
```

Comment:
You're passing the arguments, but still nothing actually receives them in the functions. This still only works because of the quirk of Bash namespacing. Please receive the arguments explicitly in the functions you've added. Look at other functions accepting positional arguments for how to do it.